src/util.ts

```diff
@@ -1080,22 +1080,15 @@ export class Util {
      * See this jsbench for why we chose this method: https://jsbench.me/r1lub4hjro
      */
     public createRange(startLine: number, startCharacter: number, endLine: number, endCharacter: number): Range {
-        //skip the cache if any number is higher than the max integer we can store for each key chunk (8191)
-        if ((startLine > 8191) || (startCharacter > 8191) || (endLine > 8191) || (endCharacter > 8191)) {
-            return {
-                start: this.createPosition(startLine, startCharacter),
-                end: this.createPosition(endLine, endCharacter)
-            };
-        }
-
         // eslint-disable-next-line no-bitwise
-        const key = (startLine << 52) + (startCharacter << 39) + (endLine << 26) + (endCharacter << 13);
+        const key = (endCharacter << 24) + (endLine << 16) + (startLine << 8) + startCharacter;
```

Comment:
Using 8 bits per number only gives us a max of 255 lines in a file or 255 characters in a line before this key overflows.

Reply:
updated